docs/how-to/hip_runtime_api/memory_management/virtual_memory.rst

```diff
@@ -128,27 +189,209 @@ synchronizes the device. This causes worse resource usage and performance.
     hipMemRelease(allocHandle);
     hipMemAddressFree(ptr, size);
 
-.. _usage_virtual_memory:
-
-Memory usage
+Virtual aliases
 ================================================================================
 
-Dynamically increase allocation size
+Virtual aliases are multiple virtual memory addresses mapping to the same
+physical memory on the GPU. This allows different threads, processes, or memory
+allocations to access shared physical memory through different virtual
+addresses on different devices.
+
+Multiple virtual memory mappings can be created using multiple calls to
+:cpp:func:`hipMemMap` on the same memory allocation.
+
+.. note::
+    
+    RDNA cards may not produce correct results, if users access two different
+    virtual addresses that map to the same physical address. In this case, the
+    L1 data caches will be incoherent due to the virtual-to-physical aliasing.
+
+    These GPUs will produce correct results if users access virtual-to-physical
+    aliases using volatile pointers.
```

Comment:
```suggestion
    RDNA cards may not produce correct results, if users access two different
    virtual addresses that map to the same physical address. In this case, the
    L1 data caches will be incoherent due to the virtual-to-physical aliasing.
    These GPUs will produce correct results if users access virtual-to-physical
    aliases using volatile pointers.
```
[NIT] Combine into one paragraph as the hanging sentence, directly follows from the previous one.